isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -1042,16 +1042,34 @@ namespace Isis {
       }
     }
 
+    // Loop over the observations to find the longest file path/name in the
+    // bunch
+    int filePadding = 0;
+
+    for (int i = 0; i < numObservations; i++) {
+      BundleImageQsp bundleImage = m_statisticsResults->observations().at(i)->at(0);
```

Comment:
You should also loop over the images in each observation. In most cases there is only 1, but observation mode can create multiple images

Reply:
This also matches the nested for loops when they are printed out.

Reply:
:+1: